src/main/scala/com/gu/acquisitionFirehoseTransformer/AcquisitionToJson.scala

```diff
@@ -25,6 +26,7 @@ object AcquisitionToJson {
       acquisition.countryCode.getOrElse(""),
       acquisition.amount,
       acquisition.currency,
-      formatter.format(new Date(timestamp))
+      formatter.format(new Date(timestamp)),
+      acquisition.campaignCode.flatMap(_.headOption).getOrElse("")
```

Comment:
to be honest - while we're here why not do both? I feel like we'll want `componentId` eventually